vars/stageStatusFilename.groovy

```diff
@@ -0,0 +1,23 @@
+// vars/stageStatusFilename.groovy
+
+import java.net.URLEncoder
+
+@NonCPS
+String urlEncode() {
```

Comment:
Maybe:

```suggestion
String urlEncodedSTAGE_NAME() {
```

Since I actually missed that this is a hard coding of url encoding of the stage name last time.  However, see below...

Reply:
Since this is a local function, does the name need to be changed?   I did not change it for the current push.